pkg/experiment/metastore/client/client.go

```diff
@@ -72,16 +52,18 @@ func dial(cfg Config, logger log.Logger) (*grpc.ClientConn, error) {
 		grpc.WithDefaultServiceConfig(grpcServiceConfig),
 		grpc.WithUnaryInterceptor(otgrpc.OpenTracingClientInterceptor(opentracing.GlobalTracer())),
 	)
-	if os.Getenv("KUBERNETES_SERVICE_HOST") != "" {
-		builder, err := NewGrpcResolverBuilder(logger, cfg.MetastoreAddress)
-		if err != nil {
-			return nil, fmt.Errorf("failed to create grpc resolver builder: %w", err)
-		}
-		options = append(options, grpc.WithResolvers(builder))
-		return grpc.Dial(builder.resolverAddrStub(), options...)
-	} else {
-		return grpc.Dial(cfg.MetastoreAddress, options...)
-	}
+	// TODO: Implement k8s grpc resolver.
+	//   Note that this may require additional permissions.
+	//   Consider: https://github.com/sercand/kuberesolver
+	//	if os.Getenv("KUBERNETES_SERVICE_HOST") != "" {
+	//		builder, err := NewGrpcResolverBuilder(logger, address)
+	//		if err != nil {
+	//			return nil, fmt.Errorf("failed to create grpc resolver builder: %w", err)
+	//		}
+	//		options = append(options, grpc.WithResolvers(builder))
+	//		return grpc.Dial(builder.resolverAddrStub(), options...)
+	//	}
+	return grpc.Dial(address, options...)
```

Comment:
Disabled it for now because of permissions issue in the local deployment – nothing bad, just didn't have time to solve. Also, stumbled upon https://github.com/sercand/kuberesolver – looks pretty good